runtime/westend/src/lib.rs

```diff
@@ -179,10 +175,16 @@ impl balances::Trait for Runtime {
 
 parameter_types! {
 	pub const TransactionByteFee: Balance = 10 * MILLICENTS;
-	// for a sane configuration, this should always be less than `AvailableBlockRatio`.
 	pub const TargetBlockFullness: Perquintill = Perquintill::from_percent(25);
 }
 
+// for a sane configuration, this should always be less than `AvailableBlockRatio`.
+const_assert!(
```

Comment:
Are we 100% sure that the compiler does not include anything from that into the final binary?

Reply:
Not by experiment, but I have asked ppl before (including you probably) and the answer was yes, we are sure. 

it was already in all of the runtimes except one of them anyhow though.

Reply:
Not by experiment, but I have asked ppl before (including you probably) and the answer was yes, we are sure. 

it was already in all of the runtimes except one of them anyhow though.